src/modules/pages/components/Website/HomePage/Hero/Hero.jsx

```diff
@@ -45,10 +46,8 @@ const Hero = () => {
   const style = headerHeight ? { paddingTop: `${headerHeight}px` } : {};
 
   const handleSubmit = useCallback((email: string) => {
-    const endpoint = `${PAGE_GET_EARLY_ACCESS}?email=${email}`;
-    if (window !== 'undefined') {
-      window.open(endpoint, '_blank');
-    }
+    // $FlowFixMe - flow type state below
+    navigate(PAGE_GET_EARLY_ACCESS, { state: { email } });
```

Comment:
This looks right but as mentioned above, `location` is always `undefined` in `EarlyAccess`.